src/pages/Documents/DropDownDocuments.tsx

```diff
@@ -67,7 +67,7 @@ const DropDown = (props: Props) => {
       setRegionAdm(roles.includes(Roles.OkrugaHead));
       setCityAdm(roles.includes(Roles.CityHead));
       setClubAdm(roles.includes(Roles.KurinHead));
-      setCanEdit(roles.includes(Roles.Admin));
+      setCanEdit(roles.includes(Roles.Admin) || roles.includes(Roles.GoverningBodyAdmin));
```

Comment:
Make it constant, not a state (in the beggining of file)
`const isAdmin = roles.includes....`

No need to use states (and agro react to rerender page).